### ISSUE TITLE
no Pause Music functionality in SoundManager

### ISSUE THREAD

**Issue by n2oula** [nicklockwood/SoundManager#9]
is there any easy way to add a pause functionality to PlayMusic?
It can be sometimes essential to be able to pause a music that's playing and resume it later on at the same timing we paused it.

**Comment by nicklockwood** [comment]
Yes. If you play your music directly using a Sound object instead of through the SoundManager interface, you can pause and resume from the point you left off.

**Comment by n2oula** [comment]
Thank you for your prompt response.
I was able to modify the SoundManager and created the Pause / Resume Music playback modules.
Tested it, it's working.

I can share the code if you want.

**Comment by Meseery** [comment]
@n2oula share the code please

**Comment by Deliaz** [comment]
@n2oula @Meseery Hi. So, guys, have you Pause implementation?
